internal/cluster/mock_cluster_api.go

```diff
@@ -6,13 +6,12 @@ package cluster
 
 import (
 	context "context"
-	reflect "reflect"
-
 	strfmt "github.com/go-openapi/strfmt"
 	gomock "github.com/golang/mock/gomock"
 	gorm "github.com/jinzhu/gorm"
 	common "github.com/openshift/assisted-service/internal/common"
 	s3wrapper "github.com/openshift/assisted-service/pkg/s3wrapper"
+	reflect "reflect"
```

Comment:
I suspect this moved here because you need to run `skipper make format`. checking.

Reply:
tested. looks like you need to correct the formatting. please run `skipper make format` and ammend.